lib/pavilion/plugins/commands/result.py

```diff
@@ -191,14 +203,16 @@ def _get_tests(self, pav_cfg, tests_arg):
         return map(int, test_list)
 
     def update_results(self, pav_cfg: dict, tests: List[TestRun],
-                       log_file: IO[str]) -> bool:
+                       log_file: IO[str], save: bool = False) -> bool:
         """Update each of the given tests with the result section from the
         current version of their configs. Then rerun result processing and
         update the results in the test object (but change nothing on disk).
 
         :param pav_cfg: The pavilion config.
         :param tests: A list of test objects to update.
         :param log_file: The logfile to log results to. May be None.
+        :param save: A bool that determines if the updated results will be
+                     saved or not. By default it is always false.
```

Comment:
You don't need to restate the type.
'Whether to save the updated results to the test's result log.' 
You should check if this also saves them to the general result log, and comment about that. I don't care if it does or not, but the argument here and in the command arguments should say.